### ISSUE TITLE
Documentation for example in user-guide refers to get_file() that does not exist

### ISSUE THREAD

**Issue by skbidiap** [bioidiap/bob.ip.flandmark#6]

**Comment by siebenkopf** [comment]
Indeed, the ``get_file`` function is a local function that is defined in the rst file:
https://raw.githubusercontent.com/bioidiap/bob.ip.flandmark/master/doc/guide.rst
It would be at least necessary to add a note about this function in the documentation, or even provide an example implementation for it.

**Comment by anjos** [comment]
This function, as shown on the source of the document (http://pythonhosted.org/bob.ip.flandmark/_sources/guide.txt), only exists inside the context of the user guide. It is supposed to be replaced by your own `get_file()` that does an equivalent job.

For your reference, here is what it does:

```python
def get_file(f):
    from os.path import join
    from pkg_resources import resource_filename
    return resource_filename('bob.ip.flandmark', join('data', f))
```

As such, I'm qualifying this as an improvement to the package rather than a bug.
